napari/layers/shapes/_shapes_utils.py

```diff
@@ -385,6 +387,41 @@ def point_to_lines(point, lines):
     return index, location
 
 
+def create_box_from_bounding(bounding_box: npt.NDArray) -> npt.NDArray:
+    """Creates the axis aligned interaction box of a bounding box
+
+    Parameters
+    ----------
+    bounding_box : np.ndarray
+        2x2 array of the bounding box. The first row is the minimum values and
+        the second row is the maximum values
+
+    Returns
+    -------
+    box : np.ndarray
+        9x2 array of vertices of the interaction box. The first 8 points are
+        the corners and midpoints of the box in clockwise order starting in the
+        upper-left corner. The last point is the center of the box
+    """
+    tl = bounding_box[(0, 0), (0, 1)]
+    br = bounding_box[(1, 1), (0, 1)]
+    tr = bounding_box[(1, 0), (0, 1)]
+    bl = bounding_box[(0, 1), (0, 1)]
+    return np.array(
```

Comment:
we really need to add a non-numba env to the test matrix... I'll make an issue

Reply:
Oh. So it's just codecov nonsense?

Reply:
ahh. It may be. It may be an artifact of try of speed-up without numba.

Reply:
No. Our test suite does not run coverage on job without numba. We should fix it